### PR TITLE
making WebVR -> WebVR resources

### PR DESCRIPTION
**Pull request by caseyyee**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -270,43 +270,29 @@ <h3 class="showcase__title">Inspirit</h3>
...
+          <li class="flex--grid--item developers__item">
+            <h2>Resources</h2>
 
-            <h4>Others</h4>
-            <p>
-              <a href="http://vizor.io">Vizor</a>,
-              <a href="http://tparisi.github.io/glam/">GLAM</a>,
-              <a href="http://www.scenevr.com/">SceneVR</a>
-            </p>
+            <h4><a href="https://aframe.io/">iswebvrready.org</a></h4>
```
Comment (caseyyee) [inline index.html]: @ngokevin ok thx

**Comment by cvan** [comment]
Can we add the a-frame logo/wordmark? It should pop. And it should be much higher up. Can we reorder?

1. Masthead, tagline
2. Get started
3. Tools + Community (perhaps add Slack slackin info in header even)
4. Showcase
5. Team
6. Legal

**Comment by caseyyee** [comment]
image

@cvan @ngokevin

**Comment by dmarcos** [comment]
Is not a section `resources` under `webvr resources`  redundant?

**Comment by caseyyee** [comment]
@dmarcos any suggestions on what we could use for text instead?   I suppose we could just remove the titling too.

**Comment by dmarcos** [comment]
`Browser support status` instead of resources? Are there going to be other things under `resources`?

**Comment by caseyyee** [comment]
Not for now.   Just quick fixes to at least get the right info up at least.

**Comment by dmarcos** [comment]
Yes, I would then probably remove the section titles then `Browsers`, `Spec` , `Resources` , `Tools`

**Comment by caseyyee** [comment]
image

**Comment by caseyyee** [comment]
header
image

**Comment by caseyyee** [comment]
image

**Comment by caseyyee** [comment]
@cvan @ngokevin @dmarcos

**Comment by cvan** [comment]
addressed naming; fixes things for now. won't push to prod until we're things look okay. small incremental changes to just fix the content. the styles can really just become minimal over time. the content's more important to be front and visible (and ideally not so much vertical scrolling necessary).

**Comment by cvan** [comment]
thanks!

**Pull request merged**